pom.xml

```diff
@@ -213,7 +215,9 @@
                                 <ts.global.s2i.quarkus.jvm.builder.image>${ts.global.s2i.quarkus.jvm.builder.image}</ts.global.s2i.quarkus.jvm.builder.image>
                                 <ts.global.s2i.quarkus.native.builder.image>${ts.global.s2i.quarkus.native.builder.image}</ts.global.s2i.quarkus.native.builder.image>
                                 <!-- Services used in test suite -->
-                                <postgresql.13.image>docker.io/library/postgres:13.8</postgresql.13.image>
```

Comment:
This change should be removed. The delition makes sense for main (we do not use that image there), but not for 2.13